dockers/docker-fpm-frr/frr/bgpd/templates/general/policies.conf.j2

```diff
@@ -3,6 +3,33 @@
 !
 !
 !
+{% if constants.bgp.allow_list is defined and constants.bgp.allow_list.enabled is defined and constants.bgp.allow_list.enabled %}
+{%   if constants.bgp.allow_list.default_action is defined and constants.bgp.allow_list.default_action.strip() == 'deny' %}
+route-map ALLOW_LIST_DEPLOYMENT_ID_0_V4 permit 65535
+  set community no-export additive
+!
+route-map ALLOW_LIST_DEPLOYMENT_ID_0_V6 permit 65535
+  set community no-export additive
+{%   else %}
+route-map ALLOW_LIST_DEPLOYMENT_ID_0_V4 permit 65535
+  set community {{ constants.bgp.allow_list.drop_community }} additive
+!
+route-map ALLOW_LIST_DEPLOYMENT_ID_0_V6 permit 65535
+  set community {{ constants.bgp.allow_list.drop_community }} additive
+{%   endif %}
+!
+route-map FROM_BGP_PEER_V4 permit 2
```

Comment:
is the order always 2, will we have a route-map inserted between the set global nexthp and this one in the future?

Reply:
The allow_list uses `ALLOW_LIST_DEPLOYMENT_ID_???_V?` to insert and remove rules.
Allow list will not touch any FROM_ or TO_ route-maps. Human operators can change anything here and we can't prevent them of doing it.